woocommerce/woocommerce
# fixed bug where adjust_download_permissions was being scheduled on va…

…riable products without downloadable variations

### All Submissions:






### Changes proposed in this Pull Request:



Closes #32316. First contribution to WooCommerce project.

### How to test the changes in this Pull Request:

1. Create a new product.
2. Select "variable product".
3. Add attributes.
4. Create variations from all attributes. Ensure none are marked as "downloadable". 
5. Edit the product and save again. 
7. Check database or Tools > Scheduled Actions to ensure no "adjust_download_permissions" were scheduled.
8. Mark one or more variations as downloadable. 
9. Edit the product and save again.
10. Check database or Tools > Scheduled Actions to ensure "adjust_download_permissions" was scheduled.

### Other information:




### FOR PR REVIEWER ONLY:
